math-libs/BLAS/CMakeLists.txt

```diff
@@ -57,16 +102,21 @@ therock_cmake_subproject_declare(hipBLASLt
     -DHIPBLASLT_ENABLE_BLIS=OFF  # TODO: Evaluate
     -DBUILD_TESTING=${THEROCK_BUILD_TESTING}
     -DHIPBLASLT_BUILD_TESTING=${THEROCK_BUILD_TESTING}
+    -DHIPBLASLT_ENABLE_ROCROLLER=ON
   CMAKE_INCLUDES
     therock_explicit_finders.cmake
   COMPILER_TOOLCHAIN
     amd-hip
   BUILD_DEPS
     hipBLAS-common
+    mxDataGenerator
     rocm-cmake
+    rocRoller
     therock-boost
+    therock-fmt
     therock-googletest
     therock-msgpack-cxx
+    therock-yaml-cpp
```

Comment:
Which of those are needed at build time in addition to rocRoller? + Let's only pull them in on Linux, see above.